indico/htdocs/js/indico/jquery/declarative.js

```diff
@@ -86,14 +99,7 @@
             }
 
             if (paramsSelector) {
-                var fieldParams = {};
-                $(paramsSelector).each(function() {
-                    if (!(this.name in fieldParams)) {
-                        fieldParams[this.name] = [];
-                    }
-                    fieldParams[this.name].push($(this).val());
-                });
-                params = $.extend({}, fieldParams, params);
+                params = $.extend({}, global.getParamsFromSelectors(paramsSelector), params);
```

Comment:
I'd rather declare it as global to make eslint happy and then call it without `global.` here (we don't do it anywhere else and it feels like doing `window.foo()` instead of just `foo()` for something not directly related to "window")